internal/configs/testdata/invalid-files/everything-is-a-plan.tf

```diff
@@ -0,0 +1,11 @@
+# experiments.EverythingIsAPlan exists but is not registered as an active (or
+# concluded) experiment, so this should fail until the experiment "gate" is
+# removed.
+terraform {
+  experiments = [everything_is_a_plan]
+}
+
+moved {
+    from = "test_instance.foo"
+    to = "test_instance.bar"
```

Comment:
Thes strings will fail later when this is enabled, since this requires static references

Reply:
ha, thank you, i'll fix that so we only get the _expected_ error